### ISSUE TITLE
🐛 Bug Report: Error 500 sorting documents by $id

### ISSUE THREAD

**Issue by gepd** [appwrite/appwrite#2722]
### 👟 Reproduction steps

List documents and order it using the following options

```js
listDocuments(collection, undefined, 1, undefined, undefined, undefined, ["$id"], ["DESC"]);
```

### 👍 Expected behavior

It should return the list of  documents order by `$id`

### 👎 Actual Behavior

I get an error 500

As a workaround I can change `$id` by `_uid` to fix this issue

```js
listDocuments(collection, undefined, 1, undefined, undefined, undefined, ["_uid"], ["DESC"]);
```

### 🎲 Appwrite version

Version 0.12.x

### 💻 Operating system

Linux

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

**Comment by Meldiron** [comment]
Created PR creating `$id` alias, similar to how there is one for `queries`: https://github.com/utopia-php/database/pull/105

Next, I will take a look at why we expose `_uid`, where else, and decide what to do about it.

**Comment by TorstenDittmann** [comment]
We just released 0.13 and this should be fixed 👍🏻 

You can find the announcement [here](https://dev.to/appwrite/announcing-appwrite-013-with-major-upgrades-to-storage-and-functions-3hpf).

Thank you a lot for reporting this 🙏🏻
